MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include README.rst LICENSE
+include LICENSE
```

Comment:
README.rst is already included by default https://packaging.python.org/guides/distributing-packages-using-setuptools/#readme-rst-readme-md, let me know if you want to be explicit about that to revert that commit